index.md

```diff
@@ -31,12 +38,28 @@ permalink: /
 7. Now you could choose additional modules, but for an easy start, we recommend using the default selection. Click on “Save and Continue” for the last time.
 8. You’re done! Your Thunder site opens and you can start the tour to show you around.
 
-## What is Thunder
-1. Detailed Feature-List
-2. Who is doing it
-3. How can I contact
-4. Slack
-5. Roadmap (only main features)
+## What is Thunder?
+### 1. Detailed Feature-List
+### 2. Who is doing this: The Thunder Core Team
+The distribution is maintained by the Thunder Core Team. The Thunder Core Team are a small number of publishing experts and computer scientists led by Ingo Rübe, CTO at Hubert Burda Media. They maintain the distribution, handle industry relations, publishers’ requests and offers as well as specialists training. They also run the Coalition’s infrastructure (like [thunder.org](http://thunder.org/)) and organize events. Plus, the Core Team decides which modules are added to or removed from the distribution. The Core Team also ensures that all basic functionality is available free of charge.
+### 3. How to reach the Thunder Core Team
```

Comment:
Question-mark at end "?". And also I would prefer full form for verbs, so to use ```We would``` instead of ```We’d```.